easybuild/tools/github.py

```diff
@@ -214,6 +214,15 @@ class GithubError(Exception):
     pass
 
 
+def get_github_account():
+    """
+    Helper method, for retrieving the GitHub account for creating PR.
+    @return: github account name
+    """
+    if build_option('github_org'):
+        return build_option('github_org')
+    return build_option('github_user')
```

Comment:
@timeu now begins the code style nitpicking... ;-)

We try to avoid having multiple `return` statements in functions, it makes it harder to grasp the code.

So, I suggest this (which is shorter too):

```python
return build_option('github_org') or build_option('github_user')
```

And now that I see it like this, maybe a function is overkill? Sorry about the back-and-forth here...